### ISSUE TITLE
[Feature Request] Increase of Resolution and Texture Filtering

### ISSUE THREAD

**Issue by Papermanzero** [devmiyax/yabause#141]
Your emulator is meanwhile in my opinion the best saturn emulator.
However I have a request concerning the upscaling and filtering.
Most of the modern emulators like mupen64plus, pcsx-reloaded, dolphin, pcsx2 etc. can increase the internal system resolution to a certain value (x2, x3, x4...) with Anti-Aliasing or without. This would be convenient for several reasons. 
The OpenGL renderer already increases the internal resolution to a certain value. However some persons tend to prefer to use the original resolution and other would like to increase the resolution even higher (with Anti-Aliasing). The demand of both groups can be satisfied, if you enable a free selection of the internal resolution scaling value.
Furthermore a filter for the textures is really necessary to play games on higher resolutions especially on HDTVs. A bilinear, 2xSai, hq4x etc. would be consequently great. At least a bilinear and 2xSai filter would already improve the textures a lot. Alternatively you can implement shader functionality (stackable shaders, like Demul or PCSX Reloaded), which will take over the filtering and the possible Anti-Aliasing.
Maybe the filters of the other open source emulators can be supportive.

**Comment by devmiyax** [comment]
Yeah, I like these kind of techs. But currently we have many stability issues I'll clean up them first.

**Comment by ADormant** [comment]
XBRZ is the best texture scaler http://ngemu.com/threads/peteopengl2tweak-tweaker-for-peteopengl2-plugin-w-gte-accuracy-hack.160319/
Things like MSAA, SSAA and bilinear/trilinear/anisotropic texture filtering would be nice too.

**Comment by ADormant** [comment]
https://github.com/simias/rustation-libretro/commit/595ba82463d200b208d4d6be27c446cfe3c179d5
https://github.com/simias/rustation-libretro/commit/518f17ce6c0ce7847db75f7d8f5840d89137c033

**Comment by Papermanzero** [comment]
Meanwhile Super xBR is the best scaler ;-)
